### PR TITLE
Renamed: Key for AdvancedManualInstaller

### LINKED ISSUE #1527: Advanced installer button just calls the normal installer rather than the advanced one

### PR DESCRIPTION
**Pull request by Sewer56**
This fixes the key from `AdvancedInstaller` to `AdvancedManualInstaller`.
The key got renamed alongside the class at some point, but it seems not all cases were renamed.

Fixes
- #1527

In simpler terms, AdvancedInstaller now shows up when you hit `Add Advanced` in the UI

### CODE REVIEW

File: src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs [hunk 1]
```diff
@@ -197,7 +197,7 @@ public LoadoutGridViewModel() : base(null!)
 
     public Task AddModAdvanced(string path)
     {
-        var installer = _provider.GetKeyedService<IModInstaller>("AdvancedInstaller");
+        var installer = _provider.GetKeyedService<IModInstaller>("AdvancedManualInstaller");
```
Comment (halgari) [inline src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs:200]: we should probably make this a constant and use that on both ends (registration and usage) so we don't get this again.

**Review by halgari — CHANGES_REQUESTED** [review]

File: src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs (hunk 1 above)
Comment (Sewer56) [reply src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs:200]: I would prefer to do that too. I actually tried doing that.

The issue is, the `DataModel` refers to `AdvancedManualInstaller` key.
Using a constant name requires that I add a dependency to the DataModel.

The alternative is I define the constant in the DataModel.

Comment (Sewer56) [reply src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs:200]: Edit: We discussed this in meeting

**Review by Al12rs — APPROVED** [review]

**Review by halgari — APPROVED** [review]
